### ISSUE TITLE
Help!!!

### ISSUE THREAD

**Issue by hmm-star** [MrShadowDev/Free-Canva-Pro-Lifetime#10]
Dear sir i have installed all the dependencies and everything. Yet when i ran the command, nothing happens. No text file appears even after waiting for long. Please upload a video tutoria if possible
Best Regards

**Comment by MrShadowDev** [comment]
Let me check it again pls!

**Comment by MrShadowDev** [comment]
u copied it pasted the link?

**Comment by MrShadowDev** [comment]
U need to enter the link on the website and after u should introduce the code

**Comment by hmm-star** [comment]
yes sir i copied and pasted the link...and it says the invite link has expired. Also i get the same code even if re-run the python file

**Comment by MrShadowDev** [comment]
> yes sir i copied and pasted the link and then entered the code...and it says the invite link has expired. Also i get the same code even if re-run the python file

let me check it again pls

**Comment by MrShadowDev** [comment]
You need to go to this [link](https://imgur.com/SrG2krV) and u enter the code on the image thats on the pastebin bro

**Comment by hmm-star** [comment]
yes dear  sir i did enter the code in the dialogue box and i got the following
<img width="781" alt="2" src="https://github.com/MrShadowDev/Free-Canva-Pro-Lifetime/assets/161000975/f5d173a6-9a06-4701-8073-7215c0322290">

**Comment by MrShadowDev** [comment]
Ok so ill need to update it then its not working anymore then it worked yesterday
